modules/aggregation-layers/src/grid-layer/grid-cell-layer.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2015 - 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import {Texture} from '@luma.gl/core';
+import {UpdateParameters, DefaultProps, Color} from '@deck.gl/core';
+import {ColumnLayer} from '@deck.gl/layers';
+import {CubeGeometry} from '@luma.gl/engine';
+import {defaultColorRange, colorRangeToTexture} from '../utils/color-utils';
+import vs from './grid-cell-layer-vertex.glsl';
+
+const defaultProps: DefaultProps<_GridCellLayerProps> = {
+  colorRange: defaultColorRange,
+  elevationRange: [0, 1000]
+};
+
+/** Proprties added by GridCellLayer. */
+export type _GridCellLayerProps = {
```

Comment:
Nit: Export without underscore in the source file and control underscore in index.ts?